src/Parser/B3dmParser.js

```diff
@@ -43,77 +40,45 @@ function filterUnsupportedSemantics(obj) {
+        gltfScene.applyMatrix4(matrixChangeUpVectorXtoZ);
     }
-    glTFLoader.setDRACOLoader(dracoLoader);
 }
 
 /**
- * Enable KTX2 decoding for gltf. This library is mandatory to load b3dm and gltf with KTX2 compression.
- *
- * The KTX2 library files are in folder itowns/examples/libs/basis/.
- * You must indicate this path when you want to enable KTX2 decoding.
- * For more information about KTX2, read /itowns/examples/libs/basis/README.md.
- *
- * @example <caption>Enable ktx2 decoder</caption>
- * // if you copied /itowns/examples/libs/draco/ to the root folder of your project,you can set the path to './'.
- * itowns.enableKtx2Loader('./', view.mainLoop.gfxEngine.renderer);
- *
- * @param {string} path path to KTX2 library folder.
- * @param {THREE.WebGLRenderer} renderer the threejs renderer
+ * @module B3dmParser
  */
-export function enableKtx2Loader(path, renderer) {
-    if (!path || !renderer) {
-        throw new Error('Path to ktx2 folder and renderer are mandatory');
-    }
-    const ktx2Loader = new KTX2Loader();
-    ktx2Loader.setTranscoderPath(path);
-    ktx2Loader.detectSupport(renderer);
-    glTFLoader.setKTX2Loader(ktx2Loader);
-}
 
 export default {
     /** Parse b3dm buffer and extract THREE.Scene and batch table
      * @param {ArrayBuffer} buffer - the b3dm buffer.
      * @param {Object} options - additional properties.
      * @param {string=} [options.gltfUpAxis='Y'] - embedded glTF model up axis.
      * @param {string} options.urlBase - the base url of the b3dm file (used to fetch textures for the embedded glTF model).
-     * @param {boolean=} [options.doNotPatchMaterial='false'] - disable patching material with logarithmic depth buffer support.
+     * @param {boolean=} [options.doNotPatchMaterial=false] - disable patching material with logarithmic depth buffer support.
      * @param {float} [options.opacity=1.0] - the b3dm opacity.
-     * @param {boolean|Material=} [options.overrideMaterials='false'] - override b3dm's embedded glTF materials. If
+     * @param {boolean=} [options.frustumCulled=false] - enable frustum culling.
+     * @param {boolean|Material=} [options.overrideMaterials=false] - override b3dm's embedded glTF materials. If
      * true, a threejs [MeshBasicMaterial](https://threejs.org/docs/index.html?q=meshbasic#api/en/materials/MeshBasicMaterial)
      * is set up. config.overrideMaterials can also be a threejs [Material](https://threejs.org/docs/index.html?q=material#api/en/materials/Material)
      * in which case it will be the material used to override.
      * @return {Promise} - a promise that resolves with an object containig a THREE.Scene (gltf) and a batch table (batchTable).
      *
      */
     parse(buffer, options) {
-        const gltfUpAxis = options.gltfUpAxis;
-        const urlBase = options.urlBase;
+        const frustumCulled = options.frustumCulled === undefined || options.frustumCulled === null ? true : !!(options.frustumCulled);
```

Comment:
A simpler way could be:
```suggestion
        const frustumCulled = options.frustumCulled ?? true;
```

Moreover, the default parameter is documented as `false`.

Reply:
Set it back to false and used a more syntetic syntax